modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="creating_machine_pools_ocm_{context}"]
+ifndef::openshift-rosa[]
+= Creating a machine pool
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+= Creating a machine pool using OpenShift Cluster Manager
+endif::openshift-rosa[]
+
+ifndef::openshift-rosa[]
+A default machine pool is created when you install an {product-title} cluster. After installation, you can create additional machine pools for your cluster by using {cluster-manager}.
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+You can create additional machine pools for your {product-title} (ROSA) cluster by using {cluster-manager}.
+endif::openshift-rosa[]
+
+[IMPORTANT]
+====
+The compute node instance types, autoscaling options, and node counts that are available to you depend on your 
+ifdef::openshift-rosa[]
+ROSA 
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+{product-title} 
+endif::[]
+subscriptions, resource quotas and deployment scenario. For more information, contact your sales representative or Red Hat support.
+====
+
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You created a ROSA cluster.
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+* You created an {product-title} cluster.
+endif::[]
+
+.Procedure
+
+. From {cluster-manager-url}, navigate to the *Clusters* page and select your cluster.
+
+. Under the *Machine pools* tab, click *Add machine pool*.
+
+. Add a *Machine pool name*.
+
+. Select a *Worker node instance type* from the drop-down menu. The instance type defines the vCPU and memory allocation for each compute node in the machine pool.
+
+. Optional: Configure autoscaling for the machine pool:
```

Comment:
Currently autoscaling only enables for all of the ROSA cluster, and some specified clusters with capability access. Should we clarify that?

Reply:
@xueli181114 I have added a note for this step in the OSD assembly, as per our Slack conversation.